yalexs/lock.py

```diff
@@ -118,6 +119,10 @@ def model(self):
     def doorbell(self) -> bool:
         return self._data["Type"] in DOORMAN_MODEL_TYPES
 
+    @cached_property
+    def unlatch(self) -> bool:
```

Comment:
Maybe `can_unlatch` or `unlatch_supported`?

Note: doorbell is probably poorly named

Reply:
Yeah, that's definitely better. Renamed it to unlatch_supported :)